### PR TITLE
Add Live Search tools feature for easy to use

### PR DESCRIPTION
**Pull request by aancw**
image

### CODE REVIEW

File: generators/voip.html.sh [hunk 1]
```diff
@@ -28,6 +28,8 @@ Packages that operate on voip programs and protocols.</p>
                 <br />
                 <p>
                     <b>Tool count:</b> <a href="">${VOIP_COUNT}</a>
+                    <input type="text" id="searchTools" onkeyup="searchTools()" placeholder="Input tool name" title="Type in a name">
```
Comment (noptrix) [inline generators/voip.html.sh:31]: double lines here and also check below and webapp.html.sh :)

**Comment by noptrix** [comment]
@aancw wow man, thanks a lot for this. just check my single comment please. we can merge it in then :)

File: generators/voip.html.sh (hunk 1 above)
Comment (aancw) [reply generators/voip.html.sh:31]: Okay, will check

**Comment by aancw** [comment]
@noptrix done

**Pull request merged**